messages/create.sandbox.md

```diff
@@ -120,6 +132,14 @@ The sandbox request configuration isn't acceptable.
 
 The poll interval (%d seconds) can't be larger that wait (%d in seconds)
 
-# error.noCloneSource
+# error.noCloneSourceName
 
 Could not find the clone sandbox name "%s" in the target org.
+
+# error.noCloneSourceId
+
+Could not find the clone sandbox id "%s" in the target org.
+
+# error.cloneFlagsConflict
+
+You can not use clone Name or Id flags and clone Name or Id fields in the definition file at the same time.
```

Comment:
I'm not sure I understand what this error message means.  Can you elaborate?  Thanks!

Reply:
Based on our DMs, it _sounds_ like this error occurs if users try to use `--source-sandbox-name` AND `--definition-file` AND the definition file contains `SourceSandboxName`.  Ditto for the ID equivalents.  And I guess this isn't allowed?  If this is indeed the situation, here's my edit of the error:

> You can't specify both the --source-sandbox-name and --definition-file flags, and also include the "SourceSandboxName" option in the definition file.  Same with the --source-id flag and "SourceId" option.  Pick one method of cloning a sandbox and try again.